lib/WP_Auth0_State_Handler.php

```diff
@@ -40,6 +44,7 @@ public function issue() {
    * @return bool
    */
   protected function store() {
+    $_COOKIE[ self::COOKIE_NAME ] = $this->uniqid;
```

Comment:
who was setting this value to `$_COOKIE` previously?

Reply:
`setcookie()` (PHP core) does the work sending the header but it doesn't set the global for use later. I need to check for that cookie later on during processing